util.go

```diff
@@ -22,13 +22,25 @@ func fileExists(fname string) bool {
 	if err != nil {
 		return false
 	}
+	// file exists but might not be readable
```

Comment:
I think it would make sense to create a separate function called `fileIsReadable` and another one wrapping both, called `fileAccessible` or something similar, returning `true` if file is both `fileExists` and `fileIsReadable`.